play.pokemonshowdown.com/src/panel-popups.tsx

```diff
@@ -368,15 +402,68 @@ class OptionsPanel extends PSRoomPanel {
 					<option value="vertical" selected={PS.prefs.onepanel === 'vertical'}>Vertical tabs</option>
 				</select></label>
 			</p>
-			<hr />
-			{PS.user.named ? <p class="buttonbar" style="text-align: right">
-				<button class="button" data-href="login"><i class="fa fa-pencil"></i> Change name</button> {}
-				<button class="button" data-cmd="/logout"><i class="fa fa-power-off"></i> Log out</button>
-			</p> : <p class="buttonbar" style="text-align: right">
-				<button class="button" data-href="login"><i class="fa fa-pencil"></i> Choose name</button>
-			</p> }
+			<p>
+				<label class="checkbox">
+					<input
+						type="checkbox"
+						name="noanim"
+						onChange={this.handleOnChange}
+						checked={PS.prefs.noanim || false}
+					></input> Disable animations</label>
```

Comment:
```suggestion
				<label class="checkbox">
					<input
						type="checkbox"
						name="noanim"
						onChange={this.handleOnChange}
						checked={PS.prefs.noanim || false}
					/> Disable animations
				</label>
```
Self-closing tags are written like this in JSX. Also PS standard is to put closing tag on the same indentation as opening tag.